cmd/guaccollect/cmd/deps_dev.go

```diff
@@ -53,6 +54,8 @@ var depsDevCmd = &cobra.Command{
 		ctx := logging.WithLogger(context.Background())
 		logger := logging.FromContext(ctx)
 
+		version.DumpVersion()
```

Comment:
i dont think we need to dump the version on every invocation. I think CLIs can have a `--version` flag which should output the version,

Reply:
The goal was to know which version of the CLI runs when the server starts. 

Right now, that isn't possible.  

That was the reason for this change.

Do you have suggestions for that?